### ISSUE TITLE
Different units in output

### ISSUE THREAD

**Issue by ccztux** [NagiosEnterprises/ncpa#171]
Hi!

There are different units in api output of this request:

/api/memory/virtual?check=true&warning=80&critical=90
```json
{
  "value": {
    "returncode": 0, 
    "stdout": "OK: percent was 9% -- available was 1898721280b -- total was 2108178432b -- free was 690806784B -- used was 1417371648B | 'percent_0'=9%;80;90;"
  }
}
````
available and total unit: b
free unit: B

**Comment by ccztux** [comment]
I think this should be unit: B instead of: b:

/api/disk/logical/|?check=true&warning=80&critical=90
```json
{
  "value": {
    "returncode": 0, 
    "stdout": "OK: used_percent was 63% -- total_size was 8318124032b -- used was 5310349312b -- free was 2578419712b | 'used_percent_0'=63%;80;90;"
  }
}
```

**Comment by jhomann** [comment]
Capital B is supposed to be for "bytes" whereas b is for "bits" ... however it has been the last few releases a bit confusing because they are not correctly labeled as such. One of the updates in 1.8 was to fix this - but it looks like it may not have updated all the spots it was supposed to.

**Comment by jhomann** [comment]
This will be fixed in 1.8.2
